oom-frontend/src/app/core/auth/auth.service.ts

```diff
@@ -0,0 +1,45 @@
+import {BehaviorSubject, Observable, of} from 'rxjs';
+import { tap } from "rxjs/operators";
+import { User } from './../../shared/models/user.model';
+import { SigninInfoDTO } from './../../shared/models/dto/SigninInfoDTO.model';
+import { Injectable } from '@angular/core';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Out-Of-Memory-Team/out-of-memory/pullRequest?prid=7533964)
```suggestion
import { Injectable } from "@angular/core";
```